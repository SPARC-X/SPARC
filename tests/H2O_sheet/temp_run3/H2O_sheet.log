The program took 593.576 s.
