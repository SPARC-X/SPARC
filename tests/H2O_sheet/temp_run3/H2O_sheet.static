***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of H:
      0.5231670889       0.7522243147       0.4999275463
      0.5231670889       0.2476307779       0.4999275463
Fractional coordinates of O:
      0.4770262760       0.4999275463       0.4999275463
Total free energy (Ha): -1.706522709731152E+01
Atomic forces (Ha/Bohr):
 -5.4950039657E-03  -2.6083965133E-02  -2.5106320483E-06
 -5.4946920686E-03   2.6081679712E-02  -2.5096073939E-06
  1.0989696034E-02   2.2854217422E-06   5.0202394422E-06
Stress (Ha/Bohr**2): 
 -6.1371957992E-03   1.3831345740E-10 
  1.3831345740E-10  -5.4755397334E-03 
Stress equiv. to all periodic (GPa): 
 -7.5234389142E+00   1.6955510005E-07 
  1.6955510005E-07  -6.7123308519E+00 
