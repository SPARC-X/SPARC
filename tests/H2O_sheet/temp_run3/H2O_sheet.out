***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:30:59 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 24 5.67 5.67 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 240 57 57
FD_ORDER: 12
BC: D P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.003674932
CHEB_DEGREE: 50
NSTATES: 9
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: H2O_sheet
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 1
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  1
Mesh spacing in x-direction        :  0.1 (Bohr)
Mesh spacing in y-direction        :  0.0994737 (Bohr)
Mesh spacing in z direction        :  0.0994737 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  H2O_sheet.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H_GGA.psp8
Pseudocharge radii of atom type 1  :  6.30 6.27 6.27 (x, y, z dir)
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  O_GGA.psp8
Pseudocharge radii of atom type 2  :  6.40 6.37 6.37 (x, y, z dir)
Number of atoms of type 2          :  1
Estimated total memory usage       :  489.86 MB
Estimated memory per processor     :  489.86 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.6423005069E+00        4.740E-01        82.653
2            -5.6836703119E+00        1.786E-01        55.038
3            -5.6875206850E+00        8.754E-02        57.491
4            -5.6883168347E+00        3.164E-02        49.416
5            -5.6883796813E+00        2.173E-02        43.562
6            -5.6884076121E+00        5.584E-03        43.943
7            -5.6884084893E+00        3.209E-03        42.542
8            -5.6884089640E+00        5.688E-04        35.667
9            -5.6884090278E+00        3.881E-04        30.637
10           -5.6884090134E+00        1.580E-04        27.828
11           -5.6884090145E+00        7.400E-05        18.606
12           -5.6884090201E+00        1.061E-05        18.245
13           -5.6884090241E+00        5.311E-06        14.424
14           -5.6884090324E+00        9.432E-07        14.759
Total number of SCF: 14    

k = [0.000, 0.000, 0.000]
Occupation of state 8 = 0.000000000000000.
Occupation of state 9 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -5.6884090324E+00 (Ha/atom)
Total free energy                  : -1.7065227097E+01 (Ha)
Band structure energy              : -4.0610689207E+00 (Ha)
Exchange correlation energy        : -4.2743872099E+00 (Ha)
Self and correction energy         : -3.6112714742E+01 (Ha)
Entropy*kb*T                       : -1.5283037721E-07 (Ha)
Fermi level                        : -1.8908684034E-01 (Ha)
RMS force                          :  2.1433457089E-02 (Ha/Bohr)
Maximum force                      :  2.6656487240E-02 (Ha/Bohr)
Time for force calculation         :  0.682 (sec)
Maximum stress                     :  6.1371957992E-03 (a.u.)
Time for stress calculation        :  2.045 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  593.576 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
